modules/cluster-samples-operator.adoc

```diff
@@ -1,19 +1,50 @@
 // Module included in the following assemblies:
 //
 // *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
 
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
 [id="cluster-samples-operator_{context}"]
-= Cluster Samples Operator
+ifdef::operator-ref[= Cluster Samples Operator]
+ifdef::cluster-caps[= OpenShift samples capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Cluster Samples Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
```

Comment:
This text reads accurate, the one thing I notice is that it should probably be in a pre-install or install instruction section.

Reply:
https://github.com/openshift/openshift-docs/pull/52965 will be moving these sections to a pre-install page after this PR has merged.